src/MMIXWare.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="MMIXware"
+   name="MMIXware License" listVersionAdded="3.22">
+      <crossRefs>
+         <crossRef>https://gitlab.lrz.de/mmix/mmixware/-/blob/master/boilerplate.w</crossRef>
+      </crossRefs>
+      <text>
+         <list>
+            <item>
```

Comment:
not sure why the list and item tags got in here, but not needed

Reply:
SPDX online tools generated that. I will remove it.